src/nvidia-cuda/devcontainer-feature.json

```diff
@@ -0,0 +1,18 @@
+{
+  "id": "nvidia-cuda",
+  "name": "NVIDIA CUDA",
+  "description": "Installs NVIDIA CUDA libraries.",
+  "version": "1.0.0",
+  "options": {
+    "install_cuda": {
+      "type": "boolean",
+      "default": true,
+      "description": "Install essential CUDA libraries"
+    },
+    "install_cudnn": {
```

Comment:
Given the discussion on https://github.com/devcontainers/features/issues/55, can we switch these options to use camelCase? We need to handle back-compat for the existing features, but since this is a new feature, we could just start with camelCase from the beginning.

Reply:
Absolutely